diff -ur ka/rust/Makefile kb/rust/Makefile
--- ka/rust/Makefile	2025-11-28 08:52:31.708518919 +0000
+++ kb/rust/Makefile	2025-11-28 08:53:53.585135337 +0000
@@ -531,6 +531,7 @@
 $(obj)/ffi.o: $(src)/ffi.rs $(obj)/compiler_builtins.o FORCE
 	+$(call if_changed_rule,rustc_library)

+$(obj)/bindings.o: private skip_gendwarfksyms := $(if $(call rustc-min-version,109100),1)
 $(obj)/bindings.o: private rustc_target_flags = --extern ffi
 $(obj)/bindings.o: $(src)/bindings/lib.rs \
     $(obj)/ffi.o \
